### ISSUE TITLE
I don't see integration to add

### ISSUE THREAD

**Issue by jendrush** [nielsfaber/scheduler-component#185]
### Checklist
- [x] I checked for similar existing issues (both open and closed) before posting.
- [x] I am running the latest version.

### Expected behavior
Successfully install the integration after following the steps in the instructions. 

### Actual behavior
After successfully installing HACS, scheduler-card, downloading the scheduler-component repository, and restarting HA, I wanted to take a step that many people forget about - add integration. Unfortunately, I don't see this integration. I tried in different browsers, on different computers, deleting the cache, restarting HA, restarting the whole system, reinstalling scheduler-card and scheduler-component. I still don't see the integration, although it is in custom_components. 
HA is relatively new to me and I am trying to learn it. I don't understand the action that makes HA able to see the integration and display it in the list to be added. I don't see any errors in the logs. 

### Steps to Reproduce
Hard o reproduce.

**Comment by MarcoSuurlant** [comment]
Is there a work around for the bug?

**Comment by jendrush** [comment]
Unfortunately, I still haven't been able to solve it. I'm betting that it might be a compatibility error. A lot of things change very quickly in Home Assistant. For me, timers are very important and after a preliminary review, this is the one that seemed to me to be most easily implemented. Unfortunately I think I will have to use another solution. I also can't quite understand why HA natively doesn't have timer support in an accessible form.

**Comment by jendrush** [comment]
I have the latest version of all Home Assistant components. I don't get a message in the logs after a reboot, so probably not everything installed as it should. I initially installed HACS, created a www folder in config, then installed the Scheduler Card via HACS, up to that point everything was fine. Further in the installation instructions it said to be sure to install scheduler-component and add integration to make the scheduler work properly. So I go into HACS, integrations, then the button on the bottom right, search for Scheduler Component and select "Download repository with HACS", select version v.3.2.7 and press "Download", it takes a moment and the window disappears. The files __init__.py, actions.py, config_flow.py, const.py, timer.py, websockets.py appear in config/custom_components/scheduler/. I am restarting HA. I go into Integrations, search for Scheduler integration, but find nothing. I clear the cache, still doesn't find it. I log in to HA in incognito mode, does not find. I change the browser, still no find. I change the computer, it does not find. I don't know what I could have done wrong, I tried to do everything according to the instructions.

**Comment by nielsfaber** [comment]
HA looks in the `custom_components` folder and checks all subdirectories for their `manifest.json` file and runs the `init.py` to have the integration started. 
When restarting HA you should see in the log a line like:
> [homeassistant.loader] We found a custom integration hacs which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant

And similar for scheduler (both are custom components).

If it doesn't happen for either HACS and scheduler, you might want to check your configuration.yaml file for the logging configuration.
Example:
```
logger:
  default: warning
  logs:
    custom_components.scheduler: debug
```

default should be set to warning to see the message printed. If you didn't configure the logger, should also use warning as log level.
You can leave out debug logging for scheduler, but it might show some useful things in case installation fails (although I don't expect much info from it).

**Comment by jendrush** [comment]
I added logging to configuration.yaml, but it didn't change anything in my logs. The message you write about I think I have only in connection with HACS:
"2022-03-01 20:36:32 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration hacs which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant"

What seems strange to me is that I don't see in /config/custom_components/scheduler the file you wrote about: manifest.json.

**Comment by nielsfaber** [comment]
I recommend deleting the scheduler folder and re-installing it.
If it fails using HACS, you can manually extract the `scheduler.zip` file from the[ latest release](https://github.com/nielsfaber/scheduler-component/releases/tag/v3.2.7).

**Comment by jendrush** [comment]
Thank you. I deleted the old files, reinstalled via HACS, but the effect was the same - lots of missing files. I deleted again, uploaded manually, restarted HA, cleared the cache and the integration appeared! From the whole situation it can be concluded that HACS somehow does not download all the files from the repository... But as I wrote, I'm just starting with HA and I don't know many of the mechanisms that govern it, even the mundane lack of files like manifest.json. Now I will know. Thanks for your help and valuable information allowing me to learn more about Home Assistant. I like to know how things work, but HA is developing so fast and a lot of information is chaotic, that it's hard to learn it thoroughly.
